pydoctor/test/testpackages/reparenting_follows_aliases/main.py

```diff
@@ -0,0 +1,3 @@
+from ._myotherthing import MyClass
```

Comment:
A docstring to explain what happens here and what was the intention to setup this test would help.